src/containers/Amendments/amendmentsTable.scss

```diff
@@ -0,0 +1,137 @@
+@import '../shared/css/variables';
+@import '../shared/css/table';
+
+.amendments-page{
+  .summary {
+    padding: 0 16px;
+    margin-top: 100px;
+
+    @include for-size(tablet-portrait-up) {
+      padding: 0;
+    }
+
+    .type {
+      display: inline-block;
+      padding-bottom: 24px;
+      color: $white;
+      font-size: 32px;
+
+      @include for-size(tablet-portrait-up) {
+        font-size: 42px;
+      }
+
+      @include bold;
+    }
+  }
+
+  .wrap {
```

Comment:
Try using `<section>` tags instead.  I think there is styling for that already that falls within these sizes.

Reply:
I think the only place where the same style is being used in `.network-page .wrap` and I don't see any `<section>` tag being used anywhere else in the code